packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -363,6 +411,113 @@ contract OPContractsManager is ISemver {
         return output;
     }
 
+    /// @notice addGameType deploys a new dispute game and links it to the DisputeGameFactory. The inputted _gameConfigs
+    /// must be added in ascending GameType order.
+    function addGameType(AddGameInput[] memory _gameConfigs) external returns (AddGameOutput[] memory) {
+        if (address(this) == address(thisOPCM)) revert OnlyDelegatecall();
+        if (_gameConfigs.length == 0) revert InvalidGameConfigs();
+
+        AddGameOutput[] memory outputs = new AddGameOutput[](_gameConfigs.length);
+        Blueprints memory bps = thisOPCM.blueprints();
+
+        // Store last game config to as an int64 so that we can ensure that the same game config is not added twice.
+        // An int64 is OK to use here since the GameType is a uint32. We start at -1 since 0 is a valid GameType.
+        int64 lastGameConfig = -1;
```

Comment:
nit, solidity is funny, it's actually marginally cheaper and simpler bytecode to just use int256, otherwise it will do bit cleaning and stuff (changing this is not a blocker to merge). Also typo
```suggestion
        // Store last game config as an int64 so that we can ensure that the same game config is not added twice.
        // An int64 is OK to use here since the GameType is a uint32. We start at -1 since 0 is a valid GameType.
        int64 lastGameConfig = -1;
```

Reply:
fixed